config/initializers/devise.rb

```diff
@@ -104,7 +104,7 @@
   # initial account confirmation) to be applied. Requires additional unconfirmed_email
   # db field (see migrations). Until confirmed new email is stored in
   # unconfirmed email column, and copied to email column on successful confirmation.
-  # config.reconfirmable = true
+  config.reconfirmable = false
```

Comment:
NOTE: this needed to be explicitly set as `true` is now the default in the latest `devise` and the tests fail due to a missing column in the model.